M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -1063,7 +1135,7 @@ isRankAtLeastSingle(ZZ, Matrix) := opts -> (n1, M0) -> (
   if (n1 > numRows M0) or (n1 > numColumns M0) then return false;
   if (M0 == 0) then return (n1 <= 0);
   if (n1 == numRows M0) and (n1 == numColumns M0) then return (rank M0 == n1);
-  if (opts.MaxMinors =!= null) then (if (opts.MaxMinors <= 0) then return (rank M0 == n1););
+  if (not (opts.MaxMinors === null)) then (if (opts.MaxMinors <= 0) then return (rank M0 == n1););
```

Comment:
Simplify to one `if` by using `and`.

Reply:
done

Reply:
Whoops, I think I caught something else.  I will push a change fixing this shortly.